azurerm/resource_arm_netapp_volume_test.go

```diff
@@ -0,0 +1,322 @@
+package azurerm
```

Comment:
can we add more tests here? covering changing the subnet, usage threshold, export policy rule etc

Reply:
fixed